packages/webapp/src/_app/eos/api.ts

```diff
@@ -16,17 +18,29 @@ interface TableResponse<T> {
     next_key: string;
 }
 
+const TableParamDefaults = {
+    scope: CONTRACT_SCOPE,
+    lowerBound: "0",
+    upperBound: null,
+    limit: 1,
+}
+
+const getTableParamsWithDefault = (options: any): TableQueryOptions => (
+    Object.assign({}, TableParamDefaults, options)
+)
+
 export const getRow = async <T>(
     table: string,
-    keyName?: string,
-    keyValue?: string
+    options?: TableQueryOptions,
 ): Promise<T | undefined> => {
-    const rows = await getTableRows(table, keyValue);
+    options = getTableParamsWithDefault(options)
+    const rows = await getTableRows(table, options)
```

Comment:
Here and below on Line 55, I'd prob just do a spread for options: `{ ...TableParamDefaults, ...options }` and do away with the `getTableParamsWithDefault` function altogether. And because the `TableParamDefaults` is a constant, I'd do all caps: `TABLE_PARAM_DEFAULTS` or whatever name makes the most sense.

const rows = await getTableRows

Reply:
@brandonfancher That's what I tried to do originally, but I found that the override order didn't match what I expected... I'll recheck those findings though.

Reply:
hm, seems to work. Apparently too much <joke drug of choice> last night? :)